### PR TITLE
add lao Language translate

### PR DESCRIPTION
**Pull request by ghost**
please add "hacktoberfest-accepted" to label to Pull request

### CODE REVIEW

File: resources/lang/lao.ini [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+flags.hollowclosed = "ໂພງທີ່ມີທາງອອກປິດ"
+flags.natural = "ຕາມທຳມະຊາດ"
+; tools
+; wand tool
+tool.wand = "ຄະທາ"
+tool.wand.lore.1 = "ຄິກຊ້າຍທີ່ບລັອກເພື່ອຕັ້ງຕຳແໜ່ງ 1 ຂອງການເລືອກ"
+tool.wand.lore.2 = "ຄິກຂວາທີ່ບລັອກເພື່ອຕັ້ງຕຳແໜ່ງ 2 ຂອງການເລືອກ"
+tool.wand.lore.3 = "ໃຊ້ຄຳສັ່ງ //togglewand ເພື່ອສະຫຼັບການເຮັດວຽກຂອງຄະທາ"
+tool.wand.disabled = "ຄະທາຖືກປິດໃຊ້ງານ ໃຊ້ຄຳສັ່ງ //togglewand ເພື່ອເປີດໃຊ້ງານ"
+tool.wand.setenabled = "ຄະທາໄດ້ຖືກ{%0}!"
```
Comment (inxomnyaa) [inline resources/lang/lao.ini]: space missing

File: resources/lang/lao.ini [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+; debug tool
+tool.debug = "ເຄື່ອງມື Debug"
```
Comment (inxomnyaa) [inline resources/lang/lao.ini:81]: Is there really no translation for this? This tool displays information about blocks

File: resources/lang/lao.ini [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+tool.debug.lore.1 = "ຄລິກຊ້າຍທີ່ບລັອກເພື່ອເບິ່ງຂໍ້ມູນ"
+tool.debug.lore.2 = "ເຊັ່ນເບິ່ງຊື່ຫຼືຄ່າຄວາມເສຍຫາຍຂອງບລັອກ"
+tool.debug.lore.3 = "ໃຊ້ຄຳສັ່ງ //toggledebug ເພື່ອສະຫຼັບການເຮັດວຽກຂອງຄະທາ"
+tool.debug.disabled = "ເຄື່ອງມື Debug ຖືກປິດໃຊ້ງານ ໃຊ້ຄຳສັ່ງ //toggledebug ເພື່ອເປີດໃຊ້ງານ"
+tool.debug.setenabled = "ເຄື່ອງ Debug ຖືກ{%0}!"
```
Comment (inxomnyaa) [inline resources/lang/lao.ini]: Another missing space

**Review by inxomnyaa — CHANGES_REQUESTED** [review]

File: resources/lang/lao.ini (hunk 2 above)
Comment (ghost) [reply resources/lang/lao.ini:81]: yes we say debug because we don't have this word in our language

File: resources/lang/lao.ini (hunk 3 above)
Comment (ghost) [reply resources/lang/lao.ini]: thanks

File: resources/lang/lao.ini (hunk 1 above)
Comment (ghost) [reply resources/lang/lao.ini]: thanks

**Review by inxomnyaa — APPROVED** [review]

**Pull request merged**